main.js

```diff
@@ -287,6 +230,14 @@ function resetGame() {
   resetButton.disabled = true;
 }
 
+function defaultSetRange() {
+  currentMin = 1;
+  currentMax = 100;
+  correctNumber = getRandomRange(currentMin, currentMax);
+  minSpan.innerText = "1";
+  maxSpan.innerText = "100";
+}
```

Comment:
We will need to add a property that will restore count guesses to 0 when game is restarted.

Reply:
Cool, can you work on that while I finish up some other things?